src/Paramore.Darker/QueryProcessor.cs

```diff
@@ -38,11 +38,11 @@ public TResult Execute<TResult>(IQuery<TResult> query)
             using (var pipelineBuilder = new PipelineBuilder<TResult>(_handlerRegistry, _handlerFactory, _decoratorFactory))
             {
                 var queryContext = CreateQueryContext();
-                var pipeline = pipelineBuilder.Build(query, queryContext);
+                var handlerToExecute = pipelineBuilder.Build(query, queryContext);
```

Comment:
Just a minor point, but would it make sense to call this `entrypoint` or similar? Just to make it clear that it's not just one handler being executed, but the whole pipeline.